.github/workflows/build.yml

```diff
@@ -50,11 +50,11 @@ jobs:
             use_sccache: false
             build_wasm: true
             enable_llvm: true
-            enable_v8: true 
+            enable_v8: false
             enable_wasmi: true
             enable_wamr: true
           - build: linux-x64
-            os: ubuntu-22.04
```

Comment:
NOTE: I don't know if this is the worfklow also used for building release artifacts, but if so we can not just do this bump because it makes the artifacts incompatible with OSes that are on an older libc.

We got plenty of complaints about that in the past.

Reply:
I see. This entails rebuilding wee8. We can either merge it like this for now and downgrade later, skip wee8 for linux-x64 until it builds as expected on older ubuntu, or just block the release until wee8 is built for the correct target.

Reply:
Note: we went for option number 3.